echidna/scripts/dump_spectra.py

```diff
@@ -0,0 +1,40 @@
+import numpy
+
+import echidna.output.store as store
+import echidna.core.fill_spectrum as fill_spectrum
+
+def read_and_dump(save_directory, paths_dict, half_life_dict):
+    """ Creates both mc and reco spectra from ROOT files, dumping them to hdf5 files                                                                                                       
+    Args:
+      save_directory (str): Path to a directory where the hdf5 files will be dumped
+      paths_dict (dict): A dictionary containing the names of background or signal source of interest 
```

Comment:
@EdLeming is there an advantage to passing dictionaries to function rather than calling it separately for each ROOT file/ntuple you want to process? When using this we will most likely be processing one file at a time, so it might be simpler to have something like:

```Python
def read_and_dump(save_directory, path, name, half_life):
```